trajectory_tracker/src/trajectory_tracker.cpp

```diff
@@ -353,18 +358,18 @@ void TrackerNode::cbOdometry(const nav_msgs::Odometry::ConstPtr& odom)
   {
     const double dt = std::min(max_dt_, (odom->header.stamp - prev_odom_stamp_).toSec());
     nav_msgs::Odometry odom_compensated = *odom;
+    tf2::Transform predict;
```

Comment:
The variable name "predict" looks ambiguous. In addition, it is not used directly in both of control() and getTrackingResult(). I think it is better to calculate `origin_raw` and `origin_yaw` in `if (predict_odom_)` block and pass them to control().

Reply:
addressed